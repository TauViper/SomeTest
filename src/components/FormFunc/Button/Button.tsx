import React, { FC } from 'react';
import SendIcon from '@mui/icons-material/Send';
import ButtonUI from '@mui/material/Button';

export const Button: FC = () => (
    <ButtonUI variant='contained' type='submit' endIcon={<SendIcon />}>
        send
    </ButtonUI>
);
