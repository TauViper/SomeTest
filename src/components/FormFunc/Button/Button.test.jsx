import React from "react";
import { Button } from "./Button";
import { render, screen } from '@testing-library/react';

describe('Button', () => {
    it('render', () => {
        render(<Button />);
    });
    it('snapshot', () => {
        const { asFragment } = render(<Button />);
        expect(asFragment()).toMatchSnapshot();
    });
});