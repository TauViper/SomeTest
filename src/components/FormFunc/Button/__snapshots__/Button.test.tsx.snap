// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button snapshot 1`] = `
<DocumentFragment>
  <button
    class="MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButtonBase-root  css-sghohy-MuiButtonBase-root-MuiButton-root"
    tabindex="0"
    type="submit"
  >
    <span
      class="MuiButton-endIcon MuiButton-iconSizeMedium css-9tj150-MuiButton-endIcon"
    >
      <svg
        aria-hidden="true"
        class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
        data-testid="SendIcon"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="M2.01 21 23 12 2.01 3 2 10l15 2-15 2z"
        />
      </svg>
    </span>
    <span
      class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
    />
  </button>
</DocumentFragment>
`;
